examples/simple-page/bemhtml-templates.js

```diff
@@ -0,0 +1,153 @@
+module.exports = function() {
+  // 1. Common HTML elements
+  // — doctype, html, head, meta, title, body
+  block('page').wrap()(function() {
+    var ctx = this.ctx;
+    var data = ctx.data;
+
+    return [
+      '<!DOCTYPE html>',
+      {
+        tag: 'html',
+        attrs: { lang: 'en-US' },
+        content: [
+          {
+            tag: 'head',
+            content: [
+              {
+                tag: 'meta',
+                attrs: { charset: 'utf-8' }
+              },
+              {
+                tag: 'title',
+                content: [
```

Comment:
I still think it should be passed via BEMTREE. Ideally BEMHTML should not know anything about `data` at all.